doc/faq.rst

```diff
@@ -1246,5 +1246,37 @@ Notes:
    :code:`RUN`) and sometimes it’s captured for internal use (e.g., many
    :code:`git(1)` invocations).
 
+.. _faq_xattrs:
+
+How do I handle extended attributes in Charliecloud?
+----------------------------------------------------
+
+As noted in section :ref:`ch-image_build-cache`, Charliecloud doesn’t support
+extended attributes (xattrs) by default. Support for xattrs  can be enabled for
+:code:`ch-image` and :code:`ch-convert` by specifying the :code:`--xattrs`
+option. This will make :code:`ch-image` save and restore xattrs via the build
```

Comment:
possibly?

```suggestion
:code:`ch-image` and :code:`ch-convert` by specifying :code:`--xattrs`
or setting :code:`$CH_XATTRS`. This will make :code:`ch-image` save and restore xattrs via the build
```